CHANGELOG.md

```diff
@@ -391,6 +391,19 @@ _Code:_
     - Improved Token Protect API
       ([#634](https://github.com/cossacklabs/themis/pull/634)).
       - Decryption no longer requires an intermediate `SecureCellData` object.
+      - `SecureCellData` can now be destructured in Kotlin
```

Comment:
```suggestion
      - `SecureCellData` can now be omitted in Kotlin
```

Reply:
hm, i might be wrong. 

i just not sure that "destructured" is a best word here, i've seen it only for javascript, but I might be wrong

Reply:
Well, “destructuring binding” a common term in programming language design, but sometimes it’s called a little different in each language. In Kotlin this is indeed called [destructuring declaration](https://kotlinlang.org/docs/reference/multi-declarations.html).

Reply:
ok, just disregard my suggestion